docs/docs/self-host/self-host-lightdash-restack.mdx

```diff
@@ -0,0 +1,74 @@
+# Self-Host Lightdash with Restack
+
+Restack is a managed service for deploying and managing open-source software.
+
+## Deploy to AWS with Restack
```

Comment:
cool, have you tried this ?

Reply:
Nope! This was already in the docs. We have some users on restock